lib/app/features/feed/create_post/views/pages/create_post_modal/components/video_preview_duration.dart

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
+import 'package:ion/app/extensions/extensions.dart';
+import 'package:ion/app/utils/date.dart';
+
+class VideoPreviewDuration extends StatelessWidget {
+  const VideoPreviewDuration({
+    required this.duration,
```

Comment:
It’s already clear and well done! I just have a small suggestion—what do you think about naming it `durationInSeconds` or perhaps accepting a `Duration` parameter to make it even more intuitive?